stacks/container-apps/scheduled-task.yml

```diff
@@ -34,6 +34,13 @@ Resources:
       Path: "/"
       ManagedPolicyArns:
         - "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceEventsRole"
+      Tags:
+        - Key: Project
+          Value: ScheduledTask
```

Comment:
You want a unique tag here?  (vs just using project "Infrastructure" for scheduled tasks)